docs/list.md

```diff
@@ -465,6 +465,13 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 
 ### T
 
+#### **TBD**
+
+- **Swag**: Exclusive swag from TBD shop for Top 20, and custom hacktoberfest swags for Top 3 contributors.
+- **Requirements**: Contribute to projects that have the hacktoberfest topic. This ensures your PR counts towards the official Hacktoberfest prizes.
+- **How to sign up**: Sign up for [developer.tbd.website](https://github.com/TBD54566975/developer.tbd.website)
+- **Notes**: Visit [TBD Hacktoberfest 2023 Guide](https://dev.to/tbdevs/preptember-diaries-gearing-up-for-hacktoberfest-2023-8o2) for more insights.
+  
```

Comment:
you said to add lines

Reply:
Hi @blindaks! This flag is because there are 2 whitespace characters on line 474. If you remove those, then CodeClimate will probably be happy. 😄 

There are some other things I would like for you to change, such as your `- **How to sign up**:` line. Please make sure that your entry is similar to others on the Swag List and ping me when you'd like me to review things again. Thanks!